cats/templates/cats/about.html

```diff
@@ -0,0 +1,11 @@
+<head>
+    <title>About</title>
+</head>
+{% extends 'base.html' %}
+{% block content %}
+    <h1>About</h1>
+    <div style="width: 500px; text-align: center; margin: auto">
```

Comment:
Instead of inline styling please use classes and style it in `<style>` tag or in external `.css` file:
```suggestion
    <div class="about-content">
```
For future - if you have nested structure for styling you can use some CSS methodology for classnames like BEM: http://getbem.com/